docs/lab/README.md

```diff
@@ -0,0 +1,133 @@
+
+At sometime in a near future AL2022 wil be released as an official Amazon Linux OS. It will be based on Fedora Linux and will have SELinux enabled by default so probably changes should be made to examples accordingly:
+[Amazon Linux 2022](https://docs.aws.amazon.com/linux/al2022/release-notes/planned-changes.html)
+
+EKS Linux AMI versions always stick with a specific kernel version that can be checked [here](https://github.com/awslabs/amazon-eks-ami/blob/master/CHANGELOG.md).
+
+These are Kernel and other relevant software versions for EKS AMI Linux from Kubernetes 1.22:
+
+**EKS AMI Linux VERSIONS** 
+|      AMI      |kubelet|      Docker     |       Kernel        |   Packer    |  containerd  |
+|:--------------|:------:|:--------------:|:-------------------:|:-----------:|-------------:|
+|1.22.9-20220629| 1.22.9 |20.10.13-2.amzn2|5.4.196-108.356.amzn2|  v20220629  |1.4.13-3.amzn2|
+|1.22.9-20220620| 1.22.9 |20.10.13-2.amzn2|5.4.196-108.356.amzn2|  v20220620  |1.4.13-3.amzn2|
+|1.22.9-20220610| 1.22.9 |20.10.13-2.amzn2|5.4.196-108.356.amzn2|  v20220610  |1.4.13-3.amzn2|
+|1.22.6-20220526| 1.22.6 |20.10.13-2.amzn2|5.4.190-107.353.amzn2|  v20220526  |1.4.13-2.amzn2.0.1|
+|1.22.6-20220523| 1.22.6 |20.10.13-2.amzn2|5.4.190-107.353.amzn2|  v20220523  |1.4.13-2.amzn2.0.1|
+|1.22.6-20220429| 1.22.6 |20.10.13-2.amzn2|5.4.188-104.359.amzn2|  v20220429  |1.4.13-2.amzn2.0.1|
+|1.22.6-20220421| 1.22.6 |20.10.13-2.amzn2|5.4.188-104.359.amzn2|  v20220421  |1.4.13-2.amzn2.0.1|
+|1.22.6-20220420| 1.22.6 |20.10.13-2.amzn2|5.4.188-104.359.amzn2|  v20220420  |1.4.13-2.amzn2.0.1|
+|1.22.6-20220406| 1.22.6 |20.10.13-2.amzn2|5.4.181-99.354.amzn2 |  v20220406  |1.4.13-2.amzn2.0.1|
+|1.22.6-20220317| 1.22.6 |20.10.7-5.amzn2 |5.4.181-99.354.amzn2 |  v20220317  |1.4.6-8.amzn2|
+
+EKS can also be used with custom node groups based on [Ubuntu](https://cloud-images.ubuntu.com/docs/aws/eks/). In this case we could use Ubuntu 18.04 or Ubuntu 22.04 in our build step at the KMM CRD as a Docker image, as most likely we will find the kernel headers package matching the nodes within the distribution repository:
+
+``
+apt update && apt-cache search linux-headers | grep aws
+``
+
+Please refer the [Ubuntu EKS](aws/ubuntu-kmm-kmod.yaml) or [Amazon Linux EKS](aws/kmm-kmod.yaml) examples for further details.
+ 
+### Google Kubernetes Engine GKE
+
+AutoPilot Clusters do limit which NodeSelectors and Linux Capabilities may be used in deployments, so deployments of KMMO in GKE platform should be done in standard clusters.
+
+In Standard GKE clusters Control Plane (master) is administered by Google so labeling and/or user workloads are not allowed in those nodes. Therefore labeling a worker node as `node-role.kubernetes.io/master=` is needed before deploying the controller manager.
```

Comment:
But KMMO is able to label worker nodes as needed, right?

Reply:
KMMO is able to label worker nodes with their kernel version as needed. But user has to add label `node-role.kubernetes.io/master ` manually first.